src/App.js

```diff
@@ -8,34 +8,31 @@ import Home from './components/Pages/Home/Home';
 import Footer from './components/Organisms/Footer/Footer';
 import Register from './components/Pages/Register/Register';
 import { Cart } from './components/Pages/Cart/Cart';
-import { GlobalContext } from './Context/globalContext';
-import { Dialog } from './components/Pages/Modal/Dialog';
-import { Route, Switch, HashRouter as Router, Redirect} from 'react-router-dom';
+import { Route, Switch, BrowserRouter as Router, Redirect} from 'react-router-dom';
+import Success from './components/Pages/Success/Success';
 
 const App =()=>{
     // For using locale in the application. t is the core translation function, which through our key value passed
     // as param fetches the related value from our languages objects file. 'i18n' holds many properties used 
     // for getting current state of langugae in the application and updating it.
     // const {t} = useTranslation();
```

Comment:
remove from here, remove commented unused code from whole project.

Reply:
Sure salil, will remove from here nd move to index.